homeassistant/components/sanix/__init__.py

```diff
@@ -0,0 +1,40 @@
+"""The Sanix integration."""
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import CONF_SERIAL_NO, CONF_TOKEN, DOMAIN
+from .coordinator import SanixCoordinator
+from .sanix import Sanix
+
+PLATFORMS: list[Platform] = [Platform.SENSOR, Platform.BINARY_SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Sanix from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
+
+    _serial_no = entry.data[CONF_SERIAL_NO]
+    _token = entry.data[CONF_TOKEN]
+
+    sanix_api = Sanix(_serial_no, _token, async_get_clientsession(hass))
+    coordinator = SanixCoordinator(hass, sanix_api)
+    hass.data[DOMAIN][entry.entry_id] = coordinator
+
+    hass.config_entries.async_update_entry(entry, unique_id=_serial_no)
```

Comment:
Why do we do this?

Reply:
Not needed. Sorry, it's my first integration, I took a pattern from other integrations and I left it by mistake.